stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -123,13 +135,200 @@ The `description` property allows you to provide additional information. It can
+components such as datasets, algorithms, software, storage or computation.
+ Developers can find development kits, governance templates and easy-to-integrate
+ applications to design their own applications and data sharing ecosystems.
+  Participants can provide any off-chain component they want to make it 
+  findable, available and interoperable with other components of the ecosystem."
+        lastElement={
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'right',
+              alignItems: 'center',
+              gap: '10px',
+              marginTop: '10px'
+            }}
+          >
+            <Typography color="inverted-text" fontSize="small" fontWeight="bold">
+              <a
+                style={{ color: 'inherit' }}
+                href="https://docs.okp4.network/docs/whitepaper/what-is-okp4-global-overview"
+                target="_blank"
+              >
+                Read more
+              </a>
+            </Typography>
+            <div style={{ display: 'flex', alignItems: 'center', gap: '4px' }}>
+              <Typography as="div" color="inverted-text" fontSize="x-small">
+                8 min
+              </Typography>
+              <Icon name="clock" size={20} invertColor />
+            </div>
+          </div>
+        }
+      />
+      <ListItem
+        title={
+          <Typography color="inverted-text" fontWeight="bold">
+            ØKP4
+          </Typography>
+        }
+        description={
+          <div style={{ display: 'grid', gap: '10px' }}>
+            <Typography color="inverted-text">Dataset</Typography>
+            <Typography color="inverted-text" fontSize="small">
+              Data Spaces, Open source, Web3 design system
+            </Typography>
+          </div>
+        }
+        firstElement={
+          <Typography
+            as="div"
+            color="inverted-text"
+            fontSize="small"
+            fontWeight="bold"
+            noWrap
+            style={{ textAlign: 'right' }}
+          >
+            PUBLIC
+          </Typography>
+        }
+        lastElement={
+          <Typography
+            as="div"
+            color="inverted-text"
+            fontSize="x-small"
+            fontWeight="x-light"
+            style={{ textAlign: 'right' }}
+          >
+            By ØKP4 - The data sharing protocol
+          </Typography>
+        }
+      />
+    </List>
+  </Story>
+</Canvas>
+
+## On click
+
+The `onClick` property makes the listItem clickable with a subtle animation and a change in cursor on hover.
+This facilitate its use by users that are used to clear visual cues as to when something is clickable.
+
+If a listItem does not have a `onClick` property it will not have this behaviour.
+
+<Canvas>
+  <Story name="On click">
+    {() => {
+      const handleClick = () => {
+        window.open('https://okp4.network/')
```

Comment:
I would avoid this behavior as the user may not want to leave the current storybook to another page... (frustration)